frontend/src/patientApp/timeOfTest/DOB.tsx

```diff
@@ -169,7 +169,7 @@ const DOB = () => {
     return <LoadingCard />;
   } else {
     return (
-      <main>
+      <div className={"prime-home flex-1"}>
```

Comment:
one thing I noticed is that the background for this view has a gray background instead of it being all white like before

**BEFORE**
<img width="1920" alt="Screen Shot 2022-10-07 at 10 27 24" src="https://user-images.githubusercontent.com/20211771/194600218-e185269a-e1b3-4b0f-8bb4-c08e2ef316e9.png">

**ON THIS BRANCH**
<img width="961" alt="Screen Shot 2022-10-07 at 11 15 35" src="https://user-images.githubusercontent.com/20211771/194600263-fa812f24-e9e7-4cdd-bc21-3e608c8b0579.png">

Reply:
This change was made on purpose because it looked weird since only this page wasn't white.

Reply:
@kenieh Can you take a look at this design change? (Moving from the grey background to the white one in the patient experience)

Reply:
Yes @georgehager that's right. 
This is what is in main today: 
https://user-images.githubusercontent.com/10108172/195122427-4ef5f939-4805-42f2-bb53-f5306fee61e6.mp4

What I am proposing is to just change the DOB background to match the rest of the pxp pages around them. I do notice now, the loading pages are back to the white background (small flash as seen in video below). If this change is OK i'll make the same change to those pages too!

https://user-images.githubusercontent.com/10108172/195128040-a14780c0-b23a-477f-8b2d-77fc9ef042c3.mp4

Reply:
So for the patient experience feels like it's really for a different set of users entirely then the rest of the page, so if it's doesn't match the whole container style that we use on the other pages I don't think that matters too much. 

To make those changes though I think it should be its own ticket.

Reply:
A new ticket has been made and in the future we can resolve those issues. #4460